src/System.Management.Automation/engine/LanguagePrimitives.cs

```diff
@@ -593,7 +593,9 @@ public static PSDataCollection<PSObject> GetPSDataCollection(object inputValue)
         /// <param name="second">Object to compare first to.</param>
```

Comment:
@daxian-dbw, your last commit had 2 failures in `PowerShell-CI-linux`
Built-in type information tests.Should have correct number of built-in type items in type table
```powershell
Expected exactly 273, but got 271.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/ETS/TypeTable.Tests.ps1: line 22
22:         $types.Count | Should -BeExactly ($isUnixStatEnabled ? 272 : 273)
```
Validate start of console host.No new assemblies are loaded
```powershell
Expected exactly $null, but got @(@{InputObject=System.Diagnostics.TraceSource.dll; SideIndicator=<=}, @{InputObject=System.Runtime.Serialization.Primitives.dll; SideIndicator=<=}).
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Host/Startup.Tests.ps1: line 125
125:         $diffs | Should -BeExactly $null
```

Reply:
@adityapatwardhan Can you please take a look at those 2 failures?
The first failure is from the test change in https://github.com/PowerShell/PowerShell/pull/11370. The changes in that PR seem not right when experimental features are disabled by default.
For the second, I remember seeing it somewhere, but couldn't remember now. Do you remember seeing it?

Reply:
For the second, I remember now we saw it in the release test run for the rc.1 release. We didn't think it was a blocker as it's OK to load less assemblies. But I don't remember if we find out what experimental feature caused this difference.